plugins/example/_multion/demo_app.py

```diff
@@ -0,0 +1,136 @@
+from fastapi import FastAPI, Request, Form, HTTPException, Query, BackgroundTasks
```

Comment:
Can you include this into plugins example as a router, not as it's own independent app?

Reply:
I've added it as a router within the plugins example.